integration-cli/docker_cli_diff_test.go

```diff
@@ -108,3 +108,10 @@ func (s *DockerSuite) TestDiffEnsureOnlyKmsgAndPtmx(c *check.C) {
 		}
 	}
 }
+
+// https://github.com/docker/docker/pull/14381#discussion_r33859347
+func (s *DockerSuite) TestDiffEmptyArgClientError(c *check.C) {
+	out, _, err := runCommandWithOutput(exec.Command(dockerBinary, "diff", ""))
```

Comment:
@LK4D4 I'm explicitly checking for non nilness errror so probably dockerCmd won't work here :/